packages/sx.js/src/types/index.ts

```diff
@@ -125,7 +125,7 @@ export type Vote = {
   space: string;
   authenticator: string;
   strategies: StrategyConfig[];
-  proposal: number;
+  proposal: number | string;
```

Comment:
Why do we change this? `offchain` network has its own types, those are for `starknet` network, we shouldn't change it.